.github/workflows/pr.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+# Controls when the action runs.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
```

Comment:
I'd recommend just configuring the repo to prevent from directly pushing on `master`. IMO now that there's 3 of us, I don't see a case where we would need to push something to master without going through a PR review process. WDYT?

Reply:
Yes, OK, that makes sense.